meta-aos-vm-common/recipes-core/images/aos-image-vm.bb

```diff
@@ -26,6 +26,7 @@ IMAGE_INSTALL:append = " \
 # AOS packages
 IMAGE_INSTALL:append = " \
     aos-deprov \
+    aos-clearhsm \
```

Comment:
As agreed we should call the script before IAM in aos-iamanager-provisioning.service. So, it should be added as rdepend in the corresponding recipe.